includes/definitions/onefs.yaml

```diff
@@ -10,3 +12,5 @@ over:
 discovery:
     - sysObjectId:
         - .1.3.6.1.4.1.12124.1
+    - sysDescr_regex:
```

Comment:
Two questions: why include the ' v8'? and why _regex what you have here is the same as a normal sysDescr